sw/cheri/tests/uart_check.cc

```diff
@@ -68,19 +25,16 @@ extern "C" void rom_loader_entry(void *rwRoot)
   Capability<void> root{rwRoot};
 
   // Create a bounded capability to the UART
-  Capability<volatile OpenTitanUart> uart = root.cast<volatile OpenTitanUart>();
-  uart.address() = 0x80100000;
-  uart.bounds()  = 0x34;
+  Capability<volatile OpenTitanUart<>> uart = root.cast<volatile OpenTitanUart<>>();
+  uart.address() = UART_ADDRESS;
+  uart.bounds()  = UART_BOUNDS;
+
+  uart->init(BAUD_RATE);
```

Comment:
How does the UART know the CPU frequency?

Reply:
The `CPU_TIMER_HZ` is common among our bare metal software and the CHERIoT RTOS. *[Implementation here.](https://github.com/microsoft/cheriot-rtos/blob/e8dfab9d3955e4a536c4d334e1d94b891e08f2cb/sdk/include/platform/sunburst/platform-uart.hh#L79)*

Reply:
Ok, so do we then still need it defined in: `sw/common/defs.h`?

Reply:
Yep we do